eng/scripts/Update-PkgVersion.ps1

```diff
@@ -91,9 +91,15 @@ if ($packageSemVer.HasValidPrereleaseLabel() -ne $true){
 }
 
 if (!$packageOldSemVer.IsPrerelease -and ($packageVersion -ne $NewVersionString)) {
+  $whitespace = $propertyGroup["Version"].PreviousSibling
```

Comment:
Don't you want to add the comment as a line above the ApiCompatVersion property? What does the xml end result look like with these changes?

Reply:
Yes, that is what it looks like:

```
    <Version>1.11.0-beta.1</Version>
    <!--The ApiCompatVersion will be updated automatically and does not need to be modified.-->
    <ApiCompatVersion>1.10.0</ApiCompatVersion>
```

Reply:
The whitespace will be whatever whitespace there is before the Version property. I'm not sure if this is what we want, as it seems like we would just want a single crlf, but this was the existing behavior already.

Reply:
I guess we don't use /r/n because of different line endings on different platforms?

Reply:
OK I understand now I believe. Yeah we couldn't really find a great way to copy the whitespace so we just copied what was above the Version property. I think that is a pretty reasonable to expect the whitespace to match.

Reply:
I think it would make more sense to copy what is after the version property.

Reply:
I think we chose before because we felt the it was more likely that there is another property before it where as the Version property could be the last property in the group and if that is the case then the space after wouldn't be correct.  However I'm ok either way as I suspect either would work in most cases.

Reply:
I'm fine with leaving it as is.